app/presenters/publishing_api/historical_account_presenter.rb

```diff
@@ -1,5 +1,7 @@
 module PublishingApi
   class HistoricalAccountPresenter
+    NUMBER_TO_PRESENT = 5
```

Comment:
Little nitpick: could this name be updated to reflect that it only refers to the related links, e.g. `NUMBER_OF_RELATED_LINKS`?